experiment/bumpmonitoring/main.go

```diff
@@ -46,19 +51,92 @@ var (
 )
 
 type options struct {
-	srcPath string
-	dstPath string
+	SrcPath string `json:"srcPath"`
+	DstPath string `json:"dstPath"`
+	bumper.Options
+}
+
+func parseOptions() (*options, error) {
+	var config string
+	var labelsOverride []string
+	var skipPullRequest bool
+
+	flag.StringSliceVar(&labelsOverride, "labels-override", nil, "Override labels to be added to PR.")
+	flag.BoolVar(&skipPullRequest, "skip-pullrequest", false, "")
+	flag.Parse()
+
+	var o options
+	data, err := ioutil.ReadFile(config)
```

Comment:
`config` is unset when used here so this will always fail.

Reply:
nice catch, fixed